README.md

```diff
@@ -1,2 +1,56 @@
-# HIPs
-Handshake Improvement Proposals
+# Handshake Improvement Proposals (HIPs)
+
+
+
+Handshake Improvement Proposals (HIPs) describe standards for the Handshake platform, including core protocol specifications, client APIs, and covenant standards.
+
+A browsable version of all current and draft HIPs can be found on [Future Site](HSD-Dev.org).
+
+# Contributing
+
+ 1. Review [HIP-1](HIPS/hip-1.md).
+ 2. Fork the repository by clicking "Fork" in the top right.
+ 3. Add your HIP to your fork of the repository. There is a [template HIP here](hip-template.md).
+ 4. Submit a Pull Request to Handshake [HIPs repository](https://github.com/handshake-org/HIPs/).
+
+Your first PR should be a first draft of the final HIP. It must meet the formatting criteria enforced by the build (largely, correct metadata in the header). An editor will manually review the first PR for a new HIP and assign it a number before merging it. Make sure you include a `discussions-to` header with the URL to a discussion forum or open GitHub issue where people can discuss the HIP as a whole.
+
+If your HIP requires images, the image files should be included in a subdirectory of the `assets` folder for that HIP as follows: `assets/hip-N` (where **N** is to be replaced with the HIP number). When linking to an image in the HIP, use relative links such as `../assets/hip-1/image.png`.
+
+Once your first PR is merged, we have a bot that helps out by automatically merging PRs to draft HIPs. For this to work, it has to be able to tell that you own the draft being edited. Make sure that the 'author' line of your HIP contains either your GitHub username or your email address inside <triangular brackets>. If you use your email address, that address must be the one publicly shown on [your GitHub profile](https://github.com/settings/profile).
+
+When you believe your HIP is mature and ready to progress past the draft phase, you should do one of two things:
+
+ - **For a Standards Track HIP of type Core**, ask to have your issue added to [the agenda of an upcoming Handshake Developer meeting](Link to Meeting Schedule), where it can be discussed for inclusion in a future hard fork. If implementers agree to include it, the HIP editors will update the state of your HIP to 'Accepted'.
```

Comment:
broken link, there is no meeting schedule. how else can we attract attention to a HIP proposal?